open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

```diff
@@ -29,6 +29,7 @@
 
     private static final long recoverySleepTimeSec = 10L;
     private static final long defaultPollTimeout   = 1000;
+    private static final long defaultMaxQueueSize = 100;

```

Comment:
Should this size come from a configuration?

Reply:
Ideally, yes.  So should the defaultPollTimeout, etc.  It seems like that can be done in a follow on PR.